src/alignment.hpp

```diff
@@ -98,6 +98,7 @@ Alignment reverse_complement_alignment(const Alignment& aln, const function<int6
 vector<Alignment> reverse_complement_alignments(const vector<Alignment>& alns, const function<int64_t(int64_t)>& node_length);
 int softclip_start(const Alignment& alignment);
 int softclip_end(const Alignment& alignment);
+int query_overlap(const Alignment& aln1, const Alignment& aln2);
```

Comment:
What is the purpose of this function? I guess getting the overlap between the two Alignments? Should it count just the outermost piece of overlap or all the overlap? How does it count indels and substitutions and things?